frontend/catalyst/qjit_device.py

```diff
@@ -191,7 +191,7 @@ def default_expand_fn(self, circuit, max_expansion=10):
             will decompose to :class:`qml.QubitUnitary <pennylane.QubitUnitary>` operations.
         3. The list of device-supported gates employed by Catalyst is currently different than
             that of the ``lightning.qubit`` device, as defined by the
-            :class:`~.pennylane_extensions.QJITDevice`.
+            :class:`~.qjit_device.QJITDevice`.
```

Comment:
For future PRs: Could we gather `preprocess` and `qjit_device` in a single folder? Maybe `device/`

Reply:
For sure, more sub-modules are incoming :)